### ISSUE TITLE
Input Norms and Channel Order for EfficientNet

### ISSUE THREAD

**Issue by ivanstepanovftw** [pytorch/vision#8232]
### 📚 The doc issue

The documentation for all pretrained models lacks clear details regarding the order of color channels for input images, as well as the specific normalization mean and standard deviation values. I am particularly looking for this information in relation to the EfficientNet model.

**Comment by ivanstepanovftw** [comment]
I assume it is RGB order, and
```python
mean = (0.485, 0.456, 0.406)
std = (0.229, 0.224, 0.225)
```

**Comment by NicolasHug** [comment]
Hi @ivanstepanovftw , yes the number of channels is RGB, it's a widely adopted convention and applies everywhere in torchvision.

You're correct about the mean and std, this is documented on each model builder's page, e.g. for efficientnet_b0: https://pytorch.org/vision/main/models/generated/torchvision.models.efficientnet_b0.html#torchvision.models.EfficientNet_B0_Weights


Note that you would be able to find that in the `.transforms()` attribute of the weights as well (EfficientNet_B0_Weights.IMAGENET1K_V1.transforms) : https://pytorch.org/vision/main/models.html#initializing-pre-trained-models
